File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/event/exception/EventOpenTimeExpiredException.java
package band.gosrock.domain.domains.event.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class EventOpenTimeExpiredException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new EventOpenTimeExpiredException();

    private EventOpenTimeExpiredException() {
        super(EventErrorCode.OPEN_TIME_EXPIRED);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/event/exception/EventOpenTimeExpiredException.kt
package band.gosrock.domain.domains.event.exception

import band.gosrock.common.exception.DuDoongCodeException

class EventOpenTimeExpiredException private constructor() : DuDoongCodeException(EventErrorCode.OPEN_TIME_EXPIRED) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = EventOpenTimeExpiredException()
    }
}
